### DISCUSSION TITLE
Link to releases not showing on github.com/geoschem/geos-chem

### DISCUSSION THREAD

**Discussion by yantosca** [community/community#70648] [Repositories]
### Select Topic Area

Bug

### Body

Hi GitHub Support!

I recently noticed that the releases tab on the front of our repository

 - https://github.com/geoschem/geos-chem

no longer displays.  This is not the case with our other repos:

- https://github.com/geoschem/GCClassic
- https://github.com/geoschem/GCHP
- https://github.com/geoschem/gcpy

etc.  Is there a way to force the releases tab to show?

Tagging @msulprizio @lizziel

**Comment by yantosca** [comment]
Also I should note we have pushed releases to geoschem/geos-chem repo:

- https://github.com/geoschem/geos-chem/releases

I saw an earlier comment about that the releases tab won't show if you only push tags w/o making a release, but this is not the case here.

**Comment by lizziel** [comment]
I also noticed that our repos that have no releases still have a releases tab. For example, https://github.com/geoschem/geos_fp.

**Comment by yantosca** [comment]
Never mind!  We found that clicking the gear icon in the right sidebar brings up a menu where you can check to include releases, packages, deployments.  In our case, releases was unchecked.  We have now checked it.
